src/Networking.ino

```diff
@@ -364,6 +368,8 @@ void sendSysInfoUDP(byte repeats)
     data[14] = Settings.Build >> 8;
     memcpy((byte *)data + 15, Settings.Name, 25);
     data[40] = NODE_TYPE_ID;
+    data[41] = lowByte(Settings.WebserverPort);
+    data[42] = highByte(Settings.WebserverPort);
```

Comment:
Not sure if on the receiving side the message is accepted if its size differs from before.
Also on the receiving end you must make sure to realize this can be missing information.
Also there is some extended data version of this message if I'm not mistaken.

Maybe we should also introduce a version/revision of this message type?

Reply:
we send here 80 bytes, and check only if 1st byte is 255 and packet len is more than 2 bytes

Reply:
But do they check for packet type based on the length of the packet?

Reply:
you mean rpieasy or older versions of espeasy?
on our side we have only check for length of packet but in fact it is always true because we send/receive always 80 bytes :)

Reply:
I thought there was some part of the p2p code that actually checks on the length of a packet.
But maybe the length is a parameter in the data itself.
Have to check the code as the correctness of my memory also has some variation.

Reply:
it is possible but not in "info" packet, this one is really simple and is used to mention to other units how to discover unit

we can leave it like that

Reply:
@TD-er in legacy code (R143) send/receive info pkt is almost the same, pkt length is always 80 bytes

Reply:
I've -by accident- come across this code myself as I needed to store node information received via another entry point and indeed it is a bit more data that's being sent.

I'm now creating a struct for it so we can just byte copy the whole struct at once and also extend with other features.

I will also add the port number as you added and maybe some more fields too.

Reply:
@TD-er  so you want to merge this and extend with your work?
be aware about endiness of ints - arduino declares cpu as bigendian so can't store it directly thats why used here lowByte/highByte and makeWord

Reply:
also think about it and we should have these structs in src/DataStructs/...

Reply:
Already have it in src/DataStructs :)

Reply:
and what about webserver port?